tensorflow/core/api_def/base_api/api_def_Cos.pbtxt

```diff
@@ -1,4 +1,15 @@
 op {
   graph_op_name: "Cos"
   summary: "Computes cos of x element-wise."
+  description: << END
```

Comment:
All of these need to look like this: `description: <<END`, *without* the space.

Reply:
Thanks a lot, @martinwicke 
I have updated the code

Reply:
Do we need to trigger the test again?